package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "npm-run-all --parallel server watch:js",
     "build:js": "webpack --config webpack.config.js --progress --profile --colors",
-    "watch:js": "npm run build:js -- --watch",
```

Comment:
Thanks for this @sudheesh001!

The extra `--` is meant to be there, it marks the end of options for build:js